accounts/views.py

```diff
@@ -28,11 +28,7 @@ def update(self, request, *args, **kwargs):
             serializer = self.get_serializer(data=request.data)
 
             if serializer.is_valid():
```

Comment:
Gdzie tutaj właściwie jest podpisanie hasła na nowe? To się jakoś automatycznie robi czy jak?